tank/test_models.py

```diff
@@ -169,18 +169,27 @@ def create_and_check_module(self, dynamic, device):
         if "winograd" in self.config["flags"]:
             shark_args.use_winograd = True
 
-        try:
-            model, func_name, inputs, golden_out = download_model(
-                self.config["model_name"],
-                tank_url=self.tank_url,
-                frontend=self.config["framework"],
-                import_args=import_config,
-            )
-        except NoImportException:
-            pytest.xfail(
-                reason=f"Artifacts for this model/config must be generated locally. Please make sure {self.config['framework']} is installed."
-            )
-
+        dl_gen_attempts = 3
```

Comment:
If we need this to be greater than 2 I feel like we have bigger problems. Are we hitting spurious failures often enough that a single retry wouldn't cover them?

Reply:
It doesn't need to be greater than 2. I'll reduce it